howso/direct/client.py

```diff
@@ -195,18 +183,13 @@ def __init__(
         self.configuration = HowsoConfiguration(
             config_path=config_path, verbose=verbose)
 
-        if howso_core is None:
-            if handle not in _core_cache:
-                _core_cache[handle] = HowsoCore(
-                    self.get_unique_handle(handle),
-                    **kwargs
-                )
-            self.howso = _core_cache[handle]
-        elif isinstance(howso_core, HowsoCore):
+        if howso_core is not None and isinstance(howso_core, HowsoCore):
```

Comment:
I feel like maybe we should retain the ValueError in the event that something is passed but it's not an instance of HowsoCore? It seems like as we go through this transition, this exception message might actually help you.

Reply:
sure, I don't mind adding that branch of logic back.